src/quantum/config.js

```diff
@@ -242,7 +242,7 @@ const PLATFORMS = [
     platform: 'fenix-p2-aarch64',
     platformFilter: {
       and: [
-        { eq: { platform: 'android-hw-p2-8-0-aarch64' } },
+        { eq: { platform: 'android-hw-p2-8-0-android-aarch64' } },
```

Comment:
I suspect this will only show the new data. Can we include both?

Reply:
@davehunt is this better?

Reply:
I don't know the syntax, but should this be an `or` instead of an `and`?

Reply:
Probably. I don't know the syntax either.

I will leave it for @arnold-iakab to give it a try.